tools/shared/src/main/scala/org/scalajs/core/tools/linker/analyzer/Analyzer.scala

```diff
@@ -165,6 +173,7 @@ private final class Analyzer(semantics: Semantics,
   private class ClassInfo(data: Infos.ClassInfo) extends Analysis.ClassInfo {
     private[this] var _linking = false
```

Comment:
Or, don't condition on `_linkingFrom` at all anymore. (why do you need the `_linkingFrom == null` check)?